packages/instant/webpack.config.js

```diff
@@ -10,6 +18,16 @@ const config = {
         library: 'zeroExInstant',
         libraryTarget: 'umd',
     },
+    plugins: [
+        new webpack.DefinePlugin({
+            'process.env': {
+                NODE_ENV: JSON.stringify(process.env.NODE_ENV),
```

Comment:
Do we need this node_env line here? I recently removed this from the website because it was causing that error that fabio had mentioned? I believe webpack will take care of it for us

Reply:
Ah you are right that it looks like this isn't needed, good call. 

Just so I know though: can you specify which error and PR you are referencing?

Reply:
https://github.com/0xProject/0x-monorepo/pull/1263

<img width="1674" alt="screen shot 2018-11-14 at 23 02 10" src="https://user-images.githubusercontent.com/1141340/48653344-6951f300-e9b9-11e8-8688-e66ed339c040.png">